src/ci/run.sh

```diff
@@ -6,6 +6,10 @@ if [ -n "$CI_JOB_NAME" ]; then
   echo "[CI_JOB_NAME=$CI_JOB_NAME]"
 fi
 
+if [ -n "$CI_JOB_DOC_URL" ]; then
```

Comment:
I added this because I want to also add the URL to the comments posted by Rust Log Analyzer. This log entry should help RLA figure out what is the documentation URL.